css/home-page-redesign.styl

```diff
@@ -40,34 +42,45 @@ GRAY = #EFF2F5
     top: 4px
 
   .standard-button
-    background: none
-    border: 4px solid #FFFFFF
+    border: 2px solid
     border-radius: 6px
-    padding: 0.75em 1.5em
+    border-color: #FFFFFF
+    background: none
+    padding: 0.75em 1.75em
     transition: background 500ms ease
-    margin-top: 50px
+    margin: 50px 10px 0 10px
     font-size: 15px
+    font-weight: bold
+    text-decoration: none
 
     @media (max-width: 900px)
       margin-top: 20px
 
-    &:hover
-      background: rgba(0, 0, 0, 0.3)
+    &:visited
+      color: #FFFFFF
+
+    &:hover,
+    &:active,
+    &:focus
+      background: #FFFFFF
+      color: DARK-BLUE-GREEN
 
   .home-link-down
+    position: absolute
```

Comment:
This is super broken in firefox now:
<img width="512" alt="link-down" src="https://cloud.githubusercontent.com/assets/122990/16456667/48ae47e0-3dde-11e6-961c-049cad15e35e.png">

Specifying `left: 0` will fix it.